netbox_pyswagger/client.py

```diff
@@ -42,21 +47,8 @@ def fetch_swagger_spec(self):
 
         self.api_version = resp.headers['API-Version']
 
-        # Fix for Netbox #1853 https://github.com/digitalocean/netbox/issues/1853
-        for path in spec['paths'].keys():
-            for method in spec['paths'][path].keys():
-
-                if 'parameters' not in spec['paths'][path][method].keys():
-                    continue
-
-                for param in spec['paths'][path][method]['parameters']:
-                    if param['in'] != 'body':
-                        continue
-                    
-                    if 'custom_fields' not in param['schema']['properties'].keys():
-                        continue
-                
-                    if param['schema']['properties']['custom_fields']['type'] == 'string':
-                        param['schema']['properties']['custom_fields']['type'] = 'object'
-                    
+        if self.api_version == "2.2":
+            from .patch_2_2 import patch_spec
+            patch_spec(spec)
```

Comment:
I think it's a good idea to keep the patch(s) separate
Another idea would be to assume that multiple patches would be applied.
Since this patch is especially for custom_fields maybe it would be better to name it `patch_custom_fields_spec`

Reply:
good point, I will update accordingly.